common-rendering/src/editorialPalette/background.ts

```diff
@@ -263,6 +320,25 @@ const avatar = (format: ArticleFormat): string => {
 			return news[500];
 	}
 };
+
+const relatedCardBylineImage = (format: ArticleFormat): string => {
+	switch (format.theme) {
+		case ArticlePillar.Opinion:
+			return opinion[400];
+		case ArticlePillar.Sport:
+			return sport[400];
+		case ArticlePillar.Culture:
+			return culture[400];
+		case ArticlePillar.Lifestyle:
+			return lifestyle[400];
+		case ArticleSpecial.SpecialReport:
+			return specialReport[500];
+		case ArticlePillar.News:
+		default:
+			return opinion[400];
```

Comment:
This function is used for in `BylineCard` component which is exclusively used for opinion pieces in the "Related content" section of articles.
I don't think labs opinion pieces exist, but I was running into an issue locally where cards from the series "[Nils Pratley on Finance](https://www.theguardian.com/business/nils-pratley-on-finance)" were being rendered with red quotation marks and and a red avatar background. I have never seen that in DCR, so I decided to default to opinion colours.
What is happening is: the CAPI response for articles in that series says that the pillar is "uk/news". 
DCR transforms that data at some point (`format.theme` is `OpinionPillar`) but that information is not available in AR because the cards are of type `RelatedItemType` which only exists in the AR model. 
In short: I have never seen opinion pieces with the news colours so I'm using opinion colours in that situation.